fairlearn/metrics/_error_plotter.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""Utility class for plotting metrics with error ranges."""
+
+from typing import Any, Dict
+
+from ._metric_frame import MetricFrame
+
+
+class ErrorPlotter:
+    """ """
+
+    def __init__(self,
+                 metric_frame: MetricFrame,
+                 error_mapping: Dict[str, Dict[str, Any]]):
+        """
+        First level keys in `error_mapping`: Metrics (str corresponding to pandas column)
+        Second level keys in `error_mapping`: "upper_bound" | "lower_bound" | "upper_error" |
+                                              "lower_error" | "symmetric_error"
+        Second level values in `error_mapping`: Error Metric (str corresponding to pandas column)
+
+        error_mapping example:
+            {
+                "Recall": {
+                    "upper_bound": "Recall upper bound",
```

Comment:
so should the value be a string here or a float? I'm confused.

Reply:
I believe that the value is the column name which contains the corresponding bound.